src/tools/wix/Xsd/wix.xsd

```diff
@@ -6861,7 +6861,7 @@
       </xs:attribute>
       <xs:attribute name="Account" type="xs:string">
         <xs:annotation>
-          <xs:documentation>The account under which to start the service. Valid only when ServiceType is ownProcess.</xs:documentation>
+          <xs:documentation>The fully qualified account name under which to start the service. This is true even for local accounts (e.g.: ".\LOCAL_ACCOUNT"). Valid only when ServiceType is ownProcess.</xs:documentation>
```

Comment:
Instead of "This is true even for local accounts..." how about "Fully qualified names must be used even for local accounts..."?